pages/api/labs/finance/questionGenerator.ts

```diff
@@ -12,7 +12,7 @@ export const generateQuestionForFinanceSkill = (skill: Skill): Question => {
     case Skill.FINANCE_BUDGET:
       return getRandomFinanceQuestion();
     case Skill.FINANCE_UNIT_PRICES:
-      const noun1 = getRandomItemFromMap(fruitsMap);
+      let noun1 = getRandomItemFromMap(fruitsMap);
```

Comment:
don't think this change is necessary. its better to use const instead of let where possible. let allows you to reassign the variable to a different value later in the code. If you use const, then you're not allowed to do that. 

Using const is better, because when you see a const, you know for sure that the variable value is never going to change.

Reply:
thank you! changed.